docs/tempo/website/api_docs/_index.md

```diff
@@ -115,6 +118,48 @@ Returns:
 By default this endpoint returns [OpenTelemetry](https://github.com/open-telemetry/opentelemetry-proto/tree/main/opentelemetry/proto/trace/v1) JSON,
 but if it can also send OpenTelemetry proto if `Accept: application/protobuf` is passed.
 
+### Search
+
+<span style="background-color:#f3f973;">This experimental endpoint is disabled by default and can be enabled via the search_enabled YAML config option.</span>
+
+Tempo's Search API finds traces based on span and process attributes (tags and values).  The API is available in the query frontend service in
+a microservices deployment, or the Tempo endpoint in a single binary deployment.  The following request is used to find traces containing spans
+from service "myservice" and the url contains "api/myapi".
+
+```
+GET /api/search?service.name=myservice&http.url=api/myapi
+```
+
+Each query parameter is of the form <name>=<value>, where <name> is the name of any span-level or process-level attribute.  The value is matched as a case-insenstive substring.  There are several reserved query parameters:
```

Comment:
Suggestion corrects spelling.

I have a further question:  what is a "reserved" query parameter? Would it clarify to say that these are defined parameters?
 
```suggestion
Each query parameter is of the form <name>=<value>, where <name> is the name of any span-level or process-level attribute.  The value is matched as a case-insensitive substring.  There are several reserved query parameters:
```

Reply:
This API was created generically, and uses query parameters of the form `name=value (&name2=value2...)` In this case there is no distinction between the generic parameters and reserved names, i.e. `limit` is interpreted as a special value, and not as a value to search for.